### ISSUE TITLE
A11y_SQLAzureDataStudio_Extension_Install-UninstallExtension_ScreenReader: Heading content not reading as heading by screen reader.

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/azuredatastudio#9411]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11yMAS; #A11yTCS; #SQL Azure Data Studio; #DesktopApp; #MacOs; #Benchmark; #A11y_SQLAzureDataStudioBenchmark; #A11ySev2; #MAS2.4.6; #MAS1.3.1; #VoiceOver;

**Environment Details:**
Version: 1.16.0-insider
Commit: 506c6a5
Date: 2020-02-17T05:43:18.399Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
Screen reader: Voice Over
MAS reference: MAS2.4.6, MAS1.3.1

**Repro Steps:**
Launch Azure Data Studio.
From the left menu, click the 'Extensions' icon to open the Extension Manager.
Choose any extension from the library.
Click the green 'Install' button to install.
Launch Voice Over screen reader and navigate to the heading 'Admin pack for SQL Server'.
Check if all headings are announced by screen reader.

**Actual:**
Heading content is not announced by Voice Over as heading to screen reader users present at the top of the screen. "Admin pack for SQL Server".

**Expected:**
Heading content should be announced as heading by Voice Over.

**User Impact:**
Screen reader users might miss the heading content present on the screen.

**Attachment link reference:**
[9411_A11y_SQLAzureDataStudio_Extension_Install-UninstallExtension_ScreenReader.zip](https://github.com/microsoft/azuredatastudio/files/4297761/9411_A11y_SQLAzureDataStudio_Extension_Install-UninstallExtension_ScreenReader.zip)

**Comment by alanrenmsft** [comment]
@v-atkulk this is an upstream issue, you can also reproduce it with VSCode, please file the issue in vscode github repo.

**Comment by hannahqin** [comment]
@v-atkulk We're marking this bug for Exclusion. Please work with @Alexandria-R or another SME to review.

This is an upstream bug with VS Code and we filed a bug in their repo here: https://github.com/microsoft/vscode/issues/94955

**Importance of Experience:** somewhat important
Extensions are a crucial part of using Azure Data Studio, but visiting the extension manager is likely not an everyday (or even every-week) action. The extension manager is where users can browse, install, uninstall, and disable extensions — which may be necessary every once in a while, but infrequently.

**Quality of Experience:** Not impacted
Since users must select "Admin Pack for SQL Server" from a list to open this page, even if the title of the newly opened page is identified as text content, the user would likely be able to deduce that the text "Admin Pack for SQL Server" is the heading, since it matches the name of the extension they just selected. The same logic applies for all extensions.

AAT Conformance Doc: https://cela-ra-aat-prod.azurewebsites.net/#/workflow/ee5c9007-47f4-4fcb-a4d7-a998a617edde?tab=default

**Comment by alanrenmsft** [comment]
@hannahqin I am closing this one since this is showing up on our s360 dashboard.

**Comment by Raghu-addepalli** [comment]
https://user-images.githubusercontent.com/91586743/218713793-ff3d4166-de15-4192-bd1a-235325bc3eb4.mov

Verified the issue Now it is not reproducing.
Git Hub Tags: #Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-SQLADS-MAC-Feb2020;
